data-prepper-plugins/opensearch-source/README.md

```diff
@@ -87,20 +87,13 @@ opensearch-source-pipeline:
       hosts: [ "https://source-cluster:9200" ]
       username: "username"
       password: "password"
-  processor:
-    - add_entries:
-        entries:
-          - key: "document_id"
-            value_expression: "getMetadata(\"document_id\")"
-          - key: "index"
-            value_expression: "getMetadata(\"index\")"
   sink:
     - opensearch:
         hosts: [ "https://sink-cluster:9200" ]
         username: "username"
         password: "password"
-        document_id_field: "document_id"
-        index: "copied-${index}"
+        document_id_field: "getMetadata(\"opensearch-document_id\")"
+        index: "${getMetadata(\"opensearch-index\"}"
```

Comment:
WHat's the difference between `index` and `document_id_field`? Is `${}` required?

Reply:
document_id_field currently only supports a single field in the Event. It does not support formatting the document id like is done in index. We could add later potentially but right now document id either supports a field in the Event or it supports an expression